<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AssetExportDialog</class>
 <widget class="QDialog" name="AssetExportDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>768</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Export QML</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QListView" name="filesView"/>
   </item>
   <item row="0" column="0">
    <widget class="QLineEdit" name="exportPathEdit"/>
   </item>
   <item row="2" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
